test/browser_test.dart

```diff
@@ -42,18 +44,42 @@ void main() {
 
       expect(browser.name, 'Chrome');
       expect(browser.isChrome, true);
+      expect(browser.isEdgeChrome, isFalse);
       expect(browser.isFirefox, false);
       expect(browser.isSafari, false);
       expect(browser.isInternetExplorer, false);
       expect(browser.version, Version(53, 0, 2785, build: '143'));
     });
 
+    test('Edge Chrome', () {
+      Browser.navigator = testChromeEdge();
+      browser = Browser.getCurrentBrowser();
+
+      expect(browser.name, 'Edge');
+      expect([browser.isChrome, browser.isEdgeChrome], everyElement(isTrue),
+          reason:
+              'Blink-based Edge browser should be detected as both Chrome and Edge '
+              'since feature support is based on Chrome, but should identify as Edge '
+              'for logging purposes');
```

Comment:
This test ensures that any pre-existing feature-based business logic keying off of `isChrome` remains unchanged for Edge browsers.